extras/compareResults.R

```diff
@@ -1,73 +1,74 @@
+#' Comparing DQD results
+#' 
+#' Use:
+#' compare_results(file_old, file_new, saving_dir)
+
 library("dplyr")
-library("jsonlite")
 library("ggplot2")
-library("plotly")
-
-# Directories
-setwd('/my/path/to/dqd/extra/scripts/')
-file_old <- "json/results/folder/file1.json"
-file_new <- "json/results/folder/file2.json"
-saving_dir <- "results_compare/"
-dir.create(file.path(getwd(), saving_dir), showWarnings = FALSE)
-
-
-# List all differences
-# ... between OLD
-result_old <- jsonlite::fromJSON(file_old)
-check_results_old <- tibble(result_old$CheckResults)
-
-# ... and NEW
-result_new <- jsonlite::fromJSON(file_new)
-check_results_new <- tibble(result_new$CheckResults)
+# Other packages used: jsonlite, plotly
 
-# ... only keep the different
-combined_results <- check_results_old %>%
-  left_join(check_results_new,
-            by=c("CHECK_NAME", "CDM_TABLE_NAME", "CDM_FIELD_NAME",
-                 "CONCEPT_ID", "UNIT_CONCEPT_ID"),
-            suffix=c(".old", ".new")) %>%
-  filter(PCT_VIOLATED_ROWS.old != PCT_VIOLATED_ROWS.new)
+compare_results <- function(jsonPath.old, jsonPath.new, saving_dir){
```

Comment:
Just to be in line with the other functions in the package
```suggestion
compareDqResults <- function(jsonPath.old, jsonPath.new, saving_dir){
```